torch/distributed/internal_rpc_utils.py

```diff
@@ -6,54 +6,111 @@
 import pickle
 import traceback
 import threading
-import sys
+import six
 
 import torch
 
+# Thread local tensor tables to store tensors while pickling torch.Tensor
+# objects
 _thread_local_tensor_tables = threading.local()
-_thread_local_tensor_tables.recv_tables = []
-_thread_local_tensor_tables.send_tables = []
 
+# This class provides serialize() and deserialize() interfaces to serialize
```

Comment:
Use python docstrings for the class and member functions instead of multiple lines starting with '#'.